pages/document-upload.tsx

```diff
@@ -133,7 +133,9 @@ export default function documentUpload({
     const uploadSuccessUI = (): JSX.Element => {
         return (
             <Wrapper session={session}>
-                <CloseButton href={`${redirect as string}`} />
+                <CloseButton
+                    href={redirect ? (redirect as string) : undefined}
```

Comment:
Does `undefined` not result in an error? If there is a case where the close button has an undefined `href` would we instead want to have a default route there vs no behaviour?

Reply:
After reading on I saw 
```
    if (!onClick && !href) {
        onClick = () => router.back();
    }
 ```
 I guess there is a default behaviour if it is undefined :)